packages-native/carousel/src/ui/styles.ts

```diff
@@ -0,0 +1,77 @@
+import { TextStyle, ViewStyle } from "react-native";
+
+export interface DotStyle extends ViewStyle {
+    inactiveOpacity?: number;
+    inactiveScale?: number;
+    inactiveColor?: string;
+}
+
+export interface ActiveDotStyle extends ViewStyle {
+    color?: string;
```

Comment:
why not use `ViewStyle.backgroundColor`?

Reply:
Because technically it is not a background color and background color is also a `string` :)

Reply:
Then extends `TextStyle` instead